backend/routes/auth.js

```diff
@@ -88,17 +88,21 @@ function login(req, res, next){
 	User.authenticate(req[C.REQ_REPO].logger, req.body.username, req.body.password).then(user => {
 
 		let responseData = { username: user.user };
-		
+
 		req[C.REQ_REPO].logger.logInfo("User is logged in", responseData);
-		
+
 		responseData.roles = user.roles;
 		responseData.flags = {};
-		
+
 		if (!user.hasReadLatestTerms()) {
```

Comment:
you don't need this if statement XD

Or is it because you don't want to include this field if you dont' want the prompt? In that case then you should proabbly revert back to the old thing lol

Reply:
I don't want to include the field, but I guess it doesn't hurt if it is included.